other/react-native/BasicAutocomplete.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable react/prop-types */
+
+import React from 'react'
+import {Text, TextInput, TouchableOpacity, View} from 'react-native'
+import Downshift from '../../dist/downshift.native.cjs'
```

Comment:
Ah, CI throws a linting error when importing a `*.cjs` file, but without specifying this, `jest-resolve` can't track this module down: https://travis-ci.org/paypal/downshift/builds/338729428#L563

Should we disable the linting rule here?

Reply:
Yeah. You could probably do the same thing at the bottom of this file as we do [at the bottom of this file](https://github.com/paypal/downshift/blob/6f9a306081e960ae707ca3925aee9de834061dda/other/misc-tests/__tests__/build.js#L61-L68) :+1: